wordpress-mobile/WordPress-iOS
# Issues/4094 reader menu tags

Refs #4094 
This is part two of the reader menu refresh. This PR reintroduces tag management. 
- Adds swipe to unfollow for tags you follow
- Adds a feature for following new tags.

To test:
- Ensure sure you can swipe to delete a tag you follow.
- Ensure default topics, lists, and the logged out tag list do not support swipe to delete.
- Ensure the confirmation prompt can be canceled. 
- Ensure that unfollowing a tag removes it from the list

Needs review: @kurzee would you be game to review?
